### PR TITLE
rearrangements in sym_wann

### PR DESCRIPTION
**Pull request by stepan-tsirkin**
some rearrangements to make code more laconic

### CODE REVIEW

**Comment by Liu-Xiaoxiong** [comment]
The parties are useless here. So I will remove it.

File: wannierberri/system/sym_wann.py [hunk 1]
```diff
@@ -69,23 +70,29 @@ def __init__(
...
-        self.proj = proj
-        self.possible_matrix_list = ['AA', 'SS', 'BB', 'CC']  #['AA','BB','CC','SS','SA','SHA','SR','SH','SHR']
-        self.matrix_list = {k: v for k, v in XX_R.items() if k in self.possible_matrix_list}
+        self.possible_matrix_list = ['Ham','AA', 'SS', 'BB', 'CC']  #['AA','BB','CC','SS','SA','SHA','SR','SH','SHR']
+        self.matrix_list = XX_R
+        for k in XX_R:
+            if k not in self.possible_matrix_list:
+                print (f"WARNING: symmetrization of matrix {k} is not implemented yet, so it will not be symmetrized, but passed as it. Use on your own risk")
+        # This is confusing, actually the I-odd vectors have "+1" here, because the minus is already in the rotation matrix
+        # but Ham is a scalar, so +1
```
Comment (Liu-Xiaoxiong) [inline wannierberri/system/sym_wann.py:85]: I will cancel them

**Review by Liu-Xiaoxiong — APPROVED** [review]

**Comment by stepan-tsirkin** [comment]
> The parties are useless here. So I will remove it. 

I do not think that you are right. Parities **are** needed here. At least because spin and position matrix element may transformed differently under the same symmetry operation, e.g in case of inversion or mirror.

Note, now the tests fail heavily with large errors. Before your commit, only the sparse tests gave an error slightly above the threshold, but still small 

For me a more clear solution is to make the cartesian rotation matrix represent only the pure rotation ( so that it's determinant equals +1) and then multiply by parity_I if the operation includes inversion.

**Comment by Liu-Xiaoxiong** [comment]
The parity should be included in the symmetry operation. The different matrices respect different parity by themselves, and don't need additional parity. For the testing, I think I should check the old test of orbital magnetic moment is right or wrong. especially the part from CC_R. Let's see, maybe the test is wrong.
________________________________

**Comment by stepan-tsirkin** [comment]
Anyway, the goal of this PR was to refactor code without changing any result, so that the old tests pass. If you think that the code was wrong, open another PR that only fixes the bug and updates the tests. Thus, the changes in code will be local and easier to judge if they are correct or not.

So far, please remove your latest commit from this PR

**Pull request merged**